### DISCUSSION TITLE
Eclipse Adoptium Uses Quarkus for high volume Java SE distribution API

### DISCUSSION THREAD

**Discussion by giscus** [quarkusio/quarkus#29145] [Quarkus Blog/Website]
# Eclipse Adoptium Uses Quarkus for high volume Java SE distribution API

Quarkus: Supersonic Subatomic Java

https://quarkus.io/blog/adoptium-customer-story/

**Comment by zdtsw** [comment]
in section https://quarkus.io/blog/adoptium-customer-story/#more-information,  link to [Adoptium API Source code] should be updated to https://github.com/adoptium/api.adoptium.net

**Reply by gsmet** [reply]
Thanks for the report. I fixed it, deployment is ongoing.
